components/date-picker/internal/DatePickerBase.cs

```diff
@@ -186,7 +186,9 @@ public override CultureInfo CultureInfo
         public string DropdownClassName { get; set; }
 
         [Parameter]
-        public string Format { get; set; }
+        public OneOf<string, string[]> Format { get; set; }
```

Comment:
`Format` was originally designed for display, and I thought it would be better to add another parameter for input formatting. I think it's something like masking #3381 #1991

Reply:
Good idea, I see you. Can you check new changes?)
I added mask for input value

Reply:
Awesome, can the infrastructure related to mask also be used for both Input and InputNumber?

Reply:
No, but I refactored helper to service. I think we can use it for other inputs